src/main/java/org/apache/cassandra/sidecar/cdc/MutationHandler.java

```diff
@@ -0,0 +1,121 @@
+package org.apache.cassandra.sidecar.cdc;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.google.inject.Inject;
+import org.apache.cassandra.db.Mutation;
+import org.apache.cassandra.db.commitlog.CommitLogDescriptor;
+import org.apache.cassandra.db.commitlog.CommitLogPosition;
+import org.apache.cassandra.db.commitlog.CommitLogReadHandler;
+import org.apache.cassandra.db.partitions.PartitionUpdate;
+import org.apache.cassandra.net.MessagingService;
+import org.apache.cassandra.sidecar.Configuration;
+import org.apache.cassandra.sidecar.cdc.output.Output;
+import org.apache.cassandra.sidecar.metrics.cdc.CDCReaderMonitor;
+/**
+ * Implements Cassandra CommitLogReadHandler, dandles mutations read from Cassandra commit logs.
+ */
+public class MutationHandler implements CommitLogReadHandler
+{
+    private static final Logger logger = LoggerFactory.getLogger(MutationHandler.class);
+    private Output output;
+    Future<CommitLogPosition> mutationFuture = null;
+    private ExecutorService executor;
+    private CDCReaderMonitor monitor;
+    private CDCBookmark bookmark;
```

Comment:
we can also add "private Configuration conf;" here and at the constructor function，we can set the conf to input configuration conf ,then we can use the configuration.

Reply:
or if we can just delete input params Configuration conf

Reply:
+1, dead code, removing it.